lib/cli/src/generate-annotations.test.ts

```diff
@@ -0,0 +1,49 @@
+import dedent from 'ts-dedent';
+import * as generateAnnotations from './generate-annotations';
+
+describe('generateAnnotations', () => {
+  describe('getAnnotationsFileContent', () => {
+    test('should fix preview path if it is relative to the same folder', () => {
+      const content = generateAnnotations.getAnnotationsFileContent(['preview.ts']);
+      expect(content).toEqual(dedent`
+        import * as projectAnnotations from "./preview.ts";
+
+        export default [projectAnnotations];
+      `);
+    });
+
+    test('should replace esm with cjs', () => {
+      const content = generateAnnotations.getAnnotationsFileContent([
+        '../node_modules/@storybook/addon-actions/dist/esm/preset/addDecorator.js',
+      ]);
+
+      expect(content).toEqual(dedent`
+        import * as addonPreset from "@storybook/addon-actions/dist/esm/preset/addDecorator.js";
+        
+        export default [addonPreset];
+      `);
+    });
```

Comment:
What's going on with this test?

Reply:
I forgot to update it! 😅  thanks for the catch